system_test/aest_sync_SUITE.erl

```diff
@@ -560,6 +582,91 @@ net_split_recovery(Cfg) ->
 
     ok.
 
+net_split_mining_power(Cfg) ->
+    SplitLength = 40,
+    SyncLength = 20,
+    ExtraLength = 3,
+
+    Net1Nodes = [net1_node1, net1_node2],
+    Net2Nodes = [net2_node1, net2_node2, net2_node3, net2_node4],
+    AllNodes = Net1Nodes ++ Net2Nodes,
+
+    setup_nodes([?NET1_NODE1, ?NET1_NODE2, ?NET2_NODE1,
+                 ?NET2_NODE2, ?NET2_NODE3, ?NET2_NODE4], Cfg),
+
+    lists:foreach(fun(N) -> start_node(N, Cfg) end, Net2Nodes),
+    lists:foreach(fun(N) -> start_node(N, Cfg) end, Net1Nodes),
+
+    TargetHeight1 = SplitLength,
+    %% Wait for one extra block for resolving potential fork caused by nodes mining distinct blocks at the same time.
+    MinedHeight1 = ExtraLength + TargetHeight1,
+    wait_for_value({height, MinedHeight1}, AllNodes,
+                   (ExtraLength + SplitLength) * ?MINING_TIMEOUT, Cfg),
+
+    Net1BlocksA = lists:foldl(fun(N, Acc) ->
+        [get_block(N, TargetHeight1) | Acc]
+    end, [], Net1Nodes),
+
+    [N1A1 | N1As] = Net1BlocksA,
+    ?assertNotEqual(undefined, N1A1),
+    lists:foreach(fun(A) -> ?assertEqual(N1A1, A) end, N1As),
+
+    Net2BlocksA = lists:foldl(fun(N, Acc) ->
+        [get_block(N, TargetHeight1) | Acc]
+    end, [], Net2Nodes),
+
+    [N2A1 | N2As] = Net2BlocksA,
+    ?assertNotEqual(undefined, N2A1),
+    lists:foreach(fun(A) -> ?assertEqual(N2A1, A) end, N2As),
+
+    %% Check that the chains are different
+    ?assertNotEqual(N1A1, N2A1),
+
+    % Check that the 6 node cluster has more mining power.
+    Net1MinedBlocks1 = node_mined_retries(Net1Nodes),
+    Net2MinedBlocks1 = node_mined_retries(Net2Nodes),
+    ?assert(Net1MinedBlocks1 < Net2MinedBlocks1),
```

Comment:
In case this happens often (as it may happen for process scheduling etc...), we may declare test case skipped rather than failing. Ok for now.